static/css/home.css

```diff
@@ -1,3 +1,141 @@
+.container.main-content-shadow {
+    margin-bottom: 0!important;
```

Comment:
Missing _space_ between `0` and `!important`, but more importantly, I'm curious why we are forcing so much `!important` code? This generally means the appropriate target wasn't used, and there's bound to be a better path to target the specific element.

Reply:
Removed the `!important` from these and seems to work fine.